File: 자릿수더하기/javascript/20221018/solution.test.js
import { solution, split, sum } from './solution';

test('solution', () => {
  expect(solution(123)).toBe(6);
});

test('split', () => {
  expect(split(123)).toEqual(['1', '2', '3']);
});

test('sum', () => {
  expect(sum(['1', '2', '3'])).toBe(6);
});

File: 이상한문자만들기/java/20221020/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class SolutionTest {

    @Test
    void split() {
        Solution solution = new Solution();

        String s = "try hello world";

        assertEquals("t", solution.splitWord(s)[0]);
        assertEquals("r", solution.splitWord(s)[1]);
        assertEquals("y", solution.splitWord(s)[2]);
    }

    @Test
    void changeToUpperCase() {
        Solution solution = new Solution();

        String s = "try hello world";
        String[] words = solution.splitWord(s);

        assertEquals("TrY HeLlO WoRlD", solution.toUpperCase(words));
    }

}
